sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.students.controller.ts

```diff
@@ -93,4 +104,44 @@ export class EducationProgramOfferingStudentsController extends BaseController {
       studyEndDate: offering.studyEndDate,
     };
   }
+
+  /**
+   * Gets the offering simplified details, not including, for instance,
+   * validations, approvals and extensive data.
+   * Useful to have an overview of the offering details, for instance,
+   * when the user needs need to have quick access to data in order to
+   * support operations like confirmation of enrolment or scholastic
+   * standing requests or offering change request.
+   * @param offeringId offering.
+   * @param purpose indicating the purpose to allow for the appropriate authorization flow to be used.
+   * @param studentUserToken student user token for authorization.
+   * @returns offering details.
+   */
+  @ApiUnauthorizedResponse({
+    description: "The student is not authorized for the provided offering.",
+  })
+  @Get("offering/:offeringId/summary-details")
+  async getOfferingSummaryDetailsById(
```

Comment:
I don't think `ById` is required in a controller name

Reply:
@ann-aot I am sorry. I didn't get this. Why should we not use `ById` here?

@dheepak-aot @andrepestana-aot @guru-aot suggestions?

Reply:
I am ok either ways.

Reply:
for the same reason, https://github.com/bcgov/SIMS/pull/2196#discussion_r1297842771. `byId` makes sense in the service layer, IMO